Library/OcBootManagementLib/DefaultEntryChoice.c

```diff
@@ -970,26 +999,51 @@ InternalRegisterBootOption (
       FreePool (BootOrder);
       return EFI_OUT_OF_RESOURCES;
     }
+
+    Option = InternalGetBoostrapOptionData (
+      &OptionSize,
+      &BootOptionIndex,
+      &CurrDevicePath,
+      &BootOrder[1],
```

Comment:
Why [1] and not [0]? Did I miss anything

Reply:
We read to "(VOID *) (BootOrder + 1)" to "pre-reserve" the first index to move Bootstrap there.